test/integration/adoption_application_reviews_test.rb

```diff
@@ -39,4 +39,19 @@ class AdoptionApplicationReviewsTest < ActionDispatch::IntegrationTest
     assert_select 'a', 'Adopter Profile'
     assert_select 'a', 'Edit Application'
   end
+
+  test "verified staff can edit an adoption application status" do
+    sign_in users(:user_two)
+
+    put "/adopter_applications/#{@adopter_application.id}",
+      params: { adopter_application:
+        {
+          status: 'under_review', notes: ''
+        }, commit: 'Save', id: @adopter_application.id
+      }
+
+    assert_response :redirect
+    follow_redirect!
+    assert_select 'div', 'Status: Under Review'
+  end
```

Comment:
Just reading your comment. Interesting. I wonder if this is one case where you might want to use a query e.g., `AdoptionApplication.find_by(@adopter_application.id)` to perform a new query and pull it from the db to check if the status has changed? If we use the id of this instance variable it should not be brittle. I think you're right in that the fixture itself is not changing. But, the record in the db has changed. The fixture is more like a seed file. So, it's the db record we want to check? That's something to test out and would be good to keep in mind in future! Learning learning....

Also, you may as well test adding notes, too. You could do that in this same test, or have a separate test making a PUT call with notes then asserting presence.

Reply:
I re-ran the CI test suite and it passed. Checking the failed run the tests passed but there was one error. Probably an issues in GitHub's CI container. So, all good now.

Reply:
This test also looks great!

Reply:
I don't mean to barge in, but oftentimes I find myself `.reload`ing fixtures when testing changes.

e.g.
```rb
@dog = Dog.first

# some changes

@dog.reload

# assert the change now

I just wanted to add this in as I came across your PR when looking through the issues!

Reply:
Thanks @Eduardo06sp that looks handy. Please feel free to share any tips whenever you feel!

Reply:
Ah that reload is the method I have been missing! That's good to know about.